core/node.rb

```diff
@@ -1,106 +1,117 @@
-require 'scanf'
-require 'yaml'
+# frozen_string_literal: true
+
 require 'ipaddress'
 require 'socket'
 require_relative 'services/shell_commands'
 require_relative 'out'
 
+# Represents single node from confugiration file
 class Node
-  attr_accessor :config # Name of stand
+  include ShellCommands
+
+  AWS_METADATA_URL = 'http://169.254.169.254/latest/meta-data'
+
+  attr_accessor :config
```

Comment:
Does write permissions are really necessary when dealing with the attributes of this class? It would be better to change them into read-only `attr_reader`.